app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author_id = current_user.id
+    @comment.post_id = params[:post_id]
```

Comment:
- [ ] since you have a `current_user` method, you should use it to create a new comment. The code should look like this:
`@comment = current_user.comments.new(comment_params)`. This way you don't need to pass the user id in line 9.